lib/screens/IngredientScreen.dart

```diff
@@ -34,14 +39,32 @@ class _IngredientsPageState extends State<IngredientsPage> {
   @override
   void initState() {
     super.initState();
+    inventoryScroll = ScrollController()..addListener(_scrollListener);
```

Comment:
yes, there needs to be two periods because the function itself doesn't actually return anything, whereas the two periods will return the new scrollcontroller.